src/sacn/mem.c

```diff
@@ -112,7 +112,9 @@ typedef struct ToEraseBuf
 
 /**************************** Private variables ******************************/
 
+#if SACN_SOURCE_ENABLED
```

Comment:
Would it be worth breaking this file out into source_mem.c, etc?  I'm not particularly a fan of lots of #ifdefs sprinkled in a codebase.

Reply:
https://github.com/ETCLabs/sACN/pull/49/commits/6755f35cff4856a60753c813fbe8e7ede409303f https://github.com/ETCLabs/sACN/pull/49/commits/8b33cfb551932af29a0e42251b81f42b2817afe9 https://github.com/ETCLabs/sACN/pull/49/commits/aa8b0328a3d56468ffe5122d5b359229022ee887

Reply:
I think there's slightly less of them in mem.c now, and they're much more isolated to the functions that reference the nonexistent variables. Let me know how it looks. I'm on the fence on source_mem.c since it wouldn't reduce the number of ifs.